engine/execution/computation/computer/computer.go

```diff
@@ -429,7 +429,14 @@ func (e *blockComputer) executeTransaction(
 		lg.Info().Msg("transaction executed successfully")
 	}
 
-	e.metrics.ExecutionTransactionExecuted(time.Since(startedAt), tx.ComputationUsed, len(tx.Events), tx.Err != nil)
+	e.metrics.ExecutionTransactionExecuted(
+		time.Since(startedAt),
+		tx.ComputationUsed,
+		memAllocAfter-memAllocBefore,
```

Comment:
how expensive is to collect these two memAllocAfter and memAllocBefore, 
was wondering if we should set a flag for them to not collect them by default if they are expensive to collect ?